.changes/v2.20.0/527-features.md

```diff
@@ -0,0 +1,3 @@
+* Added support for Defined Interfaces with client methods `VCDClient.CreateDefinedInterface`, `VCDClient.GetAllDefinedInterfaces`,
+  `VCDClient.GetDefinedInterface`, `VCDClient.GetDefinedInterfaceById` and methods to manipulate them `DefinedInterface.Update`,
+  `DefinedInterface.Delete` [GH-527]
```

Comment:
This line is coming from the previous PR and is already tuned?
https://github.com/vmware/go-vcloud-director/commit/7897340a58a8031eeb840d44028872eeb35ebb1f

Probably more of the previous PR is inside of this one?

Reply:
Ah yes! I forgot to update this one :)